FirebaseInAppMessaging/Tests/Unit/FIRIAMBookKeeperViaUserDefaultsTests.m

```diff
@@ -32,15 +33,15 @@ @interface FIRIAMBookKeeperViaUserDefaultsTests : XCTestCase
 @implementation FIRIAMBookKeeperViaUserDefaultsTests
 - (void)setUp {
   [super setUp];
-  self.userDefaultsForTesting =
-      [[NSUserDefaults alloc] initWithSuiteName:@"FIRIAMBookKeeperViaUserDefaultsTests"];
+  NSString *suiteName = [[NSString alloc]
+      initWithFormat:@"FIRIAMBookKeeperViaUserDefaultsTests-%@", [NSUUID UUID].UUIDString];
```

Comment:
Will this be unique? Would a static counter be better?

Reply:
It should be unique in most cases as I think it's rare for NSUUID to generate the same UUID more than once. That being said, I'm looking at this again and want to try something else.

Reply:
I realized that I can reset the defaults container by using `NSUserDefaults` API with the same suite name. So, I reverted this change as I no longer need to namespace them with UUIDs. I can just reset the same defaults storage bucket.